src/serializers/decoders/current/v16/decode_graph.c

```
@@ -0,0 +1,256 @@
+	// Schema
+
+	// graph name
+	char *graph_name = SerializerIO_ReadBuffer(rdb, NULL);
+
+	// each key header contains the following:
+	// #nodes, #edges, #deleted nodes, #deleted edges, #labels matrices, #relation matrices
+	uint64_t node_count         = SerializerIO_ReadUnsigned(rdb);
+	uint64_t edge_count         = SerializerIO_ReadUnsigned(rdb);
+	uint64_t deleted_node_count = SerializerIO_ReadUnsigned(rdb);
+	uint64_t deleted_edge_count = SerializerIO_ReadUnsigned(rdb);
+	uint64_t label_count        = SerializerIO_ReadUnsigned(rdb);
```

cppcheck

[error] Memory leak

(memleak)